corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/list/list_view.js

```diff
@@ -0,0 +1,37 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("SessionNavigate.AppList", function (AppList, FormplayerFrontend, Backbone, Marionette) {
+    AppList.GridRow = Marionette.ItemView.extend({
+        template: "#row-template",
+        tagName: "td",
+
+        events: {
+            "click": "rowClick",
+        },
+
+        rowClick: function (e) {
+            e.preventDefault();
+            FormplayerFrontend.trigger("app:select", this.model.attributes._id);
+        },
+    });
+
+    AppList.GridView = Marionette.CompositeView.extend({
+        tagName: "div",
+        template: "#grid-template",
+        childView: AppList.GridRow,
+        attachHtml: function (collectionView, itemView) {
+            var index = this.collection.indexOf(itemView.model);
+            // This is terrible, but not sure a better way to do this - for every 3 columns make a new row.
```

Comment:
yeah so i think the easiest way to do this is not make it a table, but rather a row with `col-sm-3` blocks for each one. it will automatically go to the next line when it's out of space. we can offline about it, but essentially we can remove this logic